src/cryptography/sign.js

```diff
@@ -24,8 +25,35 @@ const signatureHeader = createHeader('SIGNATURE');
 const secondSignatureHeader = createHeader('SECOND SIGNATURE');
 const signatureFooter = createHeader('END LISK SIGNED MESSAGE');
 
-export const signMessageWithPassphrase = (message, passphrase) => {
+const SIGNED_MESSAGE_HEADER = Buffer.from('Lisk Signed Message:\n', 'utf8');
+
+export const writeVarInt = n => {
+	let buf;
+	if (n < 253) {
```

Comment:
To the extent we write our own function here (but see comment below), I think it's much easier to understand what's going on here if we specify this in hex. E.g. `if (n < 0xfd) { ... }`.